openc3-init/plugins/packages/openc3-tool-common/src/components/FileOpenSaveDialog.vue

```diff
@@ -258,9 +284,29 @@ export default {
         }
       }
     },
+    deleteTemp: function () {
+      this.$dialog
+        .confirm(`Are you sure you want to delete all the temporary files?`, {
+          okText: 'Delete',
+          cancelText: 'Cancel',
+        })
+        .then((dialog) => {
+          return Api.delete('/script-api/scripts/temp_files')
+        })
+        .then((response) => {
+          this.loadFiles()
+        })
+        .catch((error) => {
+          this.$notify.error({
+            title: 'Error',
+            body: `Failed to remove script temporary files due to ${error}`,
+          })
+        })
+    },
     openSuccess: function () {
       // Disable the buttons because the API call can take a bit
       this.disableButtons = true
+      this.selectedFile = this.selectedFile.replace(`${TEMP_PREFIX}/`, '')
```

Comment:
Why remove the temp prefix?

Reply:
_TEMP_ is just a construct since these are at the root level. So I have to remove it so open and delete work.

Reply:
I think it should be a literal folder.  I don't like them at the root level.